web/handlers.go

```diff
@@ -268,15 +268,22 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	statusCode := strconv.Itoa(w.(*responseWriterWrapper).StatusCode())
 	if c.App.Metrics() != nil {
 		c.App.Metrics().IncrementHttpRequest()
 
 		if r.URL.Path != model.API_URL_SUFFIX+"/websocket" {
 			elapsed := float64(time.Since(now)) / float64(time.Second)
-			statusCode := strconv.Itoa(w.(*responseWriterWrapper).StatusCode())
 			c.App.Metrics().ObserveApiEndpointDuration(h.HandlerName, r.Method, statusCode, elapsed)
 		}
 	}
+
+	mlog.Debug("HTTP response ["+statusCode+"]",
```

Comment:
Is there a reason behind adding status code in 2 places?

Reply:
I was receiving go vet error about unstructured logging so I decided to include it as part of the parameters as well. In any case If we’re going to reduce it to just one I’ll leave the first one

Reply:
Yes, let's use it only as a parameter so that it can be consumed by logstash and other tools.

And avoid logging that if the status code is 0 to keep things clean.

Reply:
Ah no, I think you misunderstood. You did the opposite of what I was suggesting :P

I was suggesting 2 things:
- To have the status_code as a field in structured logging so that it can be consumed by other tools. Otherwise it will be impossible to filter by a status code, which is partially our objective.
- Append the status code to the slice of fields only if it is not 0. For example:
```
fields := []mlog.Field{.., .., ,,}
if statusCode != "0" {fields = append(fields, mlog.String(..))}
mlog.Debug(.., fields...)
```

Sorry for any confusion.

Reply:
Oh I see. Such a small PR and three times doing it wrong :facepalm:

Reply:
Done and right this time :sweat: